tests/ConfigurationTest.php

```diff
@@ -21,11 +21,5 @@ public function test()
 
         // test value from env
         $this->assertSame('Europe/Berlin', Configuration::getConfig('system', 'timezone'));
-
-        // test real values
-        $this->assertSame('file', Configuration::getConfig('cache', 'type'));
-        $this->assertSame(false, Configuration::getConfig('authentication', 'enable'));
-        $this->assertSame(true, Configuration::getConfig('admin', 'donations'));
-        $this->assertSame(1, Configuration::getConfig('error', 'report_limit'));
```

Comment:
These tests were a bit silly because they pass on a fresh env but not necessarily on a dev/prod env.